windows/src/desktop/kmshell/Makefile

```diff
@@ -17,7 +17,7 @@ icons:
 xml: sentry-init.js
     cd $(ROOT)\src\desktop\kmshell\xml
     -del /q $(ROOT)\bin\desktop\xml\*
-    copy * $(ROOT)\bin\desktop\xml\   #
```

Comment:
This `\` on the end of the copy target with a `#` to prevent make from treating it as a line extension trick doesn't work in nmake (can use `/I` to force same xcopy semantics, not necessary here with copy).